### PR TITLE
feat(home): adding zenika logo

### PR DESCRIPTION
**Pull request by mfrachet**

### CODE REVIEW

File: components/Navbar.vue [hunk 1]
```diff
@@ -27,13 +25,14 @@
 </template>
 
 <script>
+import AppLogo from "~/components/Logo";
```
Comment (l-bonanni) [inline components/Navbar.vue]: 👍

File: components/Logo.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <img :alt="alt" :src="src">
```
Comment (l-bonanni) [inline components/Logo.vue]: why not put "/znk.png" here ?

File: components/modules/Home.vue [hunk 3]
```diff
@@ -42,12 +52,41 @@ export default {
...
+    AppSubtitle
   }
 };
 </script>
 
 <style>
+.home-logo-slogan {
+  text-align: center;
+  position: absolute;
+  z-index: 1;
```
Comment (l-bonanni) [inline components/modules/Home.vue]: is z-index necessary here ? 🤔

Comment (mfrachet) [reply components/modules/Home.vue]: Yes, because of Hero ^^'

File: components/Logo.vue (hunk 2 above)
Comment (mfrachet) [reply components/Logo.vue]: In case we need to use something like this later :) it's not that much job

**Pull request merged**